alembic/versions/2024_04_25_0fda0f2746d6_add_jasen_analysis_option.py

```diff
@@ -0,0 +1,132 @@
+"""Add Jasen analysis option
+
+Revision ID: 0fda0f2746d6
+Revises: f7a1394cf7cd
+Create Date: 2024-04-25 12:17:39.250700
+
+"""
+
+from alembic import op
+from enum import StrEnum
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+from sqlalchemy.orm import declarative_base
+
+
+# revision identifiers, used by Alembic.
+revision = "0fda0f2746d6"
+down_revision = "f7a1394cf7cd"
```

Comment:
It's always good to double check the down revision with master before merging, just in case of bad luck and another revision just was merged.

Reply:
Done!